test/dynamo/test_backward_higher_order_ops.py

```diff
@@ -121,23 +121,31 @@ def fn(x, y):
                 out.backward(grad_out)
             actual = normalize_gm(graph.print_readable(False))
             self.assertEqual(x.grad, grad_out * grad_out)
-            self.assertExpectedInline(
-                actual,
-                """\
+            if backend == ["aot_eager", "inductor"]:
```

Comment:
```suggestion
            if backend in ["aot_eager", "inductor"]:
```
I think this code is unreachable as written?